src/module/actor/data-model-classes/data-model-character-encumbrance-disabled.ts

```diff
@@ -1,27 +1,41 @@
-import OseDataModelCharacterEncumbrance, {CharacterEncumbrance} from './data-model-character-encumbrance';
+/**
+ * @file A class representing the "Disabled" encumbrance scheme;
+ *       we aren't tracking carry weight here.
+ */
+import OseDataModelCharacterEncumbrance, {
+  CharacterEncumbrance,
+} from "./data-model-character-encumbrance";
+
 // import { OSE } from '../../config';
 
 /**
  * @todo Add template path for encumbrance bar
  * @todo Add template path for inventory item row
  */
-export default class OseDataModelCharacterEncumbranceDisabled extends OseDataModelCharacterEncumbrance implements CharacterEncumbrance {
-	static templateEncumbranceBar = '';
-	static templateInventoryRow = '';
-	
-	/**
-	 * The machine-readable label for this encumbrance scheme
-	 */
-	static type = 'disabled';
-	/**
-	 * The human-readable label for this encumbrance scheme
-	 */
-	static localizedLabel = 'OSE.Setting.EncumbranceDisabled';
-	
-	
-	constructor() {
-		super(OseDataModelCharacterEncumbranceDisabled.type)
+export default class OseDataModelCharacterEncumbranceDisabled
+  extends OseDataModelCharacterEncumbrance
+  implements CharacterEncumbrance
+{
+  static templateEncumbranceBar = "";
+
+  static templateInventoryRow = "";
+
+  /**
+   * The machine-readable label for this encumbrance scheme
+   */
+  static type = "disabled";
+
+  /**
+   * The human-readable label for this encumbrance scheme
+   */
+  static localizedLabel = "OSE.Setting.EncumbranceDisabled";
+
+  constructor() {
+    super(OseDataModelCharacterEncumbranceDisabled.type);
   }
 
-  get value(): number { return 0 }
-}
\ No newline at end of file
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
I don't understand this one

Reply:
The `class-methods-use-this` rule makes it so that any class method must reference `this` or be a `static` member of the class. I think the idea is, if it doesn't use `this` and needs to be publicly available, it can be a static member or it can be exported separately.

Reply:
is there a reason we don't declare these as static?

Reply:
Some of them could be static. This specific line cannot, as it'll rely on `this` in other encumbrance schemes